autotest.pm

```diff
@@ -45,7 +45,7 @@ sub loadtest {
     my $category = $1;
     my $name     = $2;
     my $test;
-    my $fullname = "$category-$name";
+    my $fullname = "$category-$name-$arg";
```

Comment:
no, as the webui is not seeing this variable. This is only used to e.g. identify the SKIPTO and such. And as such it actually creates a problem. an empty arg shouldn't appear as $name- but as $name (leave alone you will get undefined warnings as it stands)